packages/client-core/package.json

```diff
@@ -77,7 +77,7 @@
     "nipplejs": "0.9.0",
     "rc-dock": "3.0.19",
     "rc-slider": "9.7.2",
-    "react": "17.0.2",
+    "react": "^17.0.0",
```

Comment:
Set this back to what it was, we're setting specific version numbers for all node_modules rather than using the '^' to let it download the latest minor version.